suitepicon/suitepicon-glyphs.scss

```diff
@@ -1,9 +1,10 @@
 @font-face {
     font-family: "suitepicon";
-    src: url("./suitepicon.eot?10a3bdc3407a37027a5802596e4e4ad7?#iefix") format("embedded-opentype"),
-url("./suitepicon.woff?10a3bdc3407a37027a5802596e4e4ad7") format("woff"),
-url("./suitepicon.ttf?10a3bdc3407a37027a5802596e4e4ad7") format("truetype"),
-url("./suitepicon.svg?10a3bdc3407a37027a5802596e4e4ad7#suitepicon") format("svg");
+    src: url("./suitepicon.eot?c5c75207c67cc2dea1c80710b12877b0?#iefix") format("embedded-opentype"),
+url("./suitepicon.woff2?c5c75207c67cc2dea1c80710b12877b0") format("woff2"),
+url("./suitepicon.woff?c5c75207c67cc2dea1c80710b12877b0") format("woff"),
+url("./suitepicon.ttf?c5c75207c67cc2dea1c80710b12877b0") format("truetype"),
+url("./suitepicon.svg?c5c75207c67cc2dea1c80710b12877b0#suitepicon") format("svg");
 }
 
 i {
```

Comment:
Please remove the "i" selector. As it will break the other fonts. A good way to check this is to go into the accounts module and click on an email address. Look at the tinemce editor. Make sure you can see the bold italic and underline buttons